src/components/button/favorite-card/favorite-card.css

```diff
@@ -0,0 +1,63 @@
+@import url("https://fonts.googleapis.com/css2?family=Signika&display=swap");
+
+.boxes {
+  display: flex;
+  gap: 1rem;
+}
+
+.favorite-card {
+  background-color: #fff2f0;
+  border-radius: 2em;
+  width: 8.25rem;
+  height: 9rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  font-family: Signika;
+  font-size: 1.0625rem;
+  padding-top: 1.5rem;
+}
+
+.favorite-card svg {
+  padding-bottom: 0.75rem;
```

Comment:
You can use gap: 0.75rem; in your favorite card-class... So you don't need a new class

Reply:
perfect! thank you!